en/mavgen_c/README.md

```diff
@@ -59,16 +60,164 @@ When compiling the project, we recommend that you specify the top-level output d
 $ gcc ... -I generated/include -I generated/include/common ...
 ```
 
-## Multiple Streams ("channels") {#streams}
+## Multiple Streams ("channels") {#channels}
```

Comment:
FYI, this section could do with a little more. I don't find this explanation usefully explains when I would use a new channel, and when not. Are there examples of what make sense as a channel?